src/main/java/fi/metatavu/kuntaapi/server/index/IndexRemoveContact.java

```diff
@@ -0,0 +1,25 @@
+package fi.metatavu.kuntaapi.server.index;
+
+public class IndexRemoveContact implements IndexRemove {
+
+  private String contactId;
+
+  @Override
+  public String getId() {
+    return contactId;
+  }
+
+  @Override
+  public String getType() {
+    return "contact";
+  }
+  
+  public void setContactId(String contactId) {
+    this.contactId = contactId;
+  }
+  
+  public String getContactId() {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Update this method so that its implementation is not identical to "getId" on line 8. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS4144)